Frontend/src/components/video/UIKitVideoPlayer.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useState, useEffect } from "react";
+import AgoraUIKit, { layout } from "agora-react-uikit";
+import axios from "axios";
+import { Button } from "@chakra-ui/react";
+
+const SERVER_URL = process.env.SERVER3_URL || "http://localhost:3500";
```

Comment:
Can add both localhost URL and deployed host URL to .env file,  have another env variable to select the URL
something along the lines of below:

In `.env` file:
```
ENV=DEV
DEV_SERVER_URL=http://localhost:3500
PROD_SERVER_URL=
```
you can leave prod url empty

```suggestion
const SERVER_URL = process.env.ENV == 'DEV' 
  ? process.env.DEV_SERVER_URL 
  : process.env.ENV == 'PROD'
      ? process.env.PROD_SERVER_URL 
      : "http://localhost:3500"
```